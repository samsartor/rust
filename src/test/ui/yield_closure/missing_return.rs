#![feature(generators, yield_closures)]

fn main() {
    let _ = || {
        yield 0;
    };
}
