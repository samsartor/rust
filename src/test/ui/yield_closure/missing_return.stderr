error[E0308]: foobar
  --> $DIR/missing_return.rs:4:16
   |
LL |       let _ = || {
   |  ________________^
LL | |         yield 0;
LL | |     };
   | |_____^ expected integer, found `()`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
